packages/math-input/src/components/keypad/button-assets.tsx

```diff
@@ -486,7 +478,97 @@ export default function ButtonAsset({id}: Props): React.ReactElement {
                 </svg>
             );
 
-        default:
+        /**
+         * ANYTHING BELOW IS NOT YET CORRECTLY HANDLED
+         */
+
+        case "PI":
+            //TODO(NickR): use correct font, size, and color for this. It's not an SVG asset
+            return <span>pi</span>;
+        case "X":
+            //TODO(NickR): use correct font, size, and color for this. It's not an SVG asset
+            return <span>x</span>;
```

Comment:
Well, this one has a value for its case, and the rest do not. That's why it's not with the others. Otherwise I'd have to implement the placeholder twice or many times instead of falling through.